tests/e2e_runner.js

```diff
@@ -102,6 +107,10 @@ testsToRun.reduce((previous, relativeName) => {
     currentFileName = relativeName.replace(/\.ts$/, '');
     const start = +new Date();
 
+    if(argv.skip && skips.some(skip => currentFileName.includes(skip))) {
```

Comment:
I thought @hansl uses different flags for these.

Overall, that's as far as I looked into the tests. Needs Hans deeper eye, for me I can't tell, but have no problems too.